status/format.go

```diff
@@ -170,19 +178,23 @@ func FormatDetails(taskRun *helpers.TaskRun) (string, error) {
 	}
 
 	if result == nil {
-		return "", nil
+		var emoji string
+		taskSucceededReason := taskRun.GetStatusCondition("Succeeded").GetReason()
```

Comment:
Suggestion: maybe we can use the `apis.ConditionSucceeded` constant instead of the hardcoded `"Succeeded"` string?

Reply:
Added the recommended conditionSucceeded, let me know if that fulfills your requirements.